pkg/runtime/typescript.go

```diff
@@ -75,6 +75,9 @@ func (t *typescript) FunctionDockerfile(funcCtxDir, version, provider string, w
 		return err
 	}
 
+	tsconfig := `'{"compilerOptions":{"esModuleInterop":true,"target":"es2015","moduleResolution":"node"}}'`
+	buildstage.Run(dockerfile.RunOptions{Command: []string{"echo", tsconfig, ">", "tsconfig.json"}})
```

Comment:
We probably want to conditionally create this if we can (similar to the way we're using `yarn import`)

Reply:
good idea, now done..